node/node.go

```diff
@@ -90,3 +92,33 @@ func (s *Service) Delete(ctx context.Context, name string) error {
 	}
 	return nil
 }
+
+// DeleteCollection deletes all nodes.
```

Comment:
One nits:
Please change other services doc as well

```suggestion
// DeleteCollection deletes Nodes according to the list options.
// And it also deletes Pods scheduled on those Nodes.
```

Reply:
Thanks. I fixed it.